frontend/src/scenes/persons/Persons.tsx

```diff
@@ -25,6 +25,7 @@ export function Persons({ cohort }: PersonsProps = {}): JSX.Element {
     const personsLogicProps: PersonLogicProps = { cohort: cohort?.id, syncWithUrl: !cohort }
     const { loadPersons, setListFilters } = useActions(personsLogic(personsLogicProps))
     const { persons, listFilters, personsLoading } = useValues(personsLogic(personsLogicProps))
+    const personHref = cohort?.id ? `/api/cohort/${cohort.id}/persons.csv?` : `/api/person.csv?${toParams(listFilters)}`
```

Comment:
Can we use the [api methods](https://github.com/PostHog/posthog/blob/a9ad5a437d2751735c050c577a143135bae793cb/frontend/src/lib/api.ts#L203-L204) here to generate this api endpoint? Ideally we'd want to avoid raw url creation.

Reply:
goood call